benchmarks/lockhammer/src/lockhammer.c

```diff
@@ -265,7 +278,7 @@ void* hmr(void *ptr)
     }
     else {
         /* Calculate affinity mask for my core and set affinity */
-        CPU_SET(((mycore >> 1)) + ((ncores >> 1) * (mycore & 1)), &affin_mask);
+        CPU_SET(((mycore * ncores / ileave) % ncores + (mycore / ileave)), &affin_mask);
```

Comment:
Hi Lucas,
Can you add a comment here about the interleaving assumption is 0 - (numcores / numthreads) is thread 0 on each physical core, (numcores / numthreads) - 2 * (numcores / numthreads) is thread 1 on each core etc?